sandbox-backend/src/main/java/com/exadel/sandbox/team5/dao/OrderDAO.java

```diff
@@ -58,18 +31,39 @@ SELECT COUNT(o.id) FROM `order` o
     List<Pair> getAllOrdersForTags();
 
     @Query(value = """
+
+            SELECT o.employeePromocode
+            FROM `order` o
+            WHERE o.employeePromocode=(:uuid);
+            """, nativeQuery = true)
+    String getEmployeePromocodeByUUID(@Param("uuid") String uuid);
+
             SELECT new com.exadel.sandbox.team5.util.Pair(c.name, COUNT(o.id))
             FROM Order o
                 JOIN o.discount d
                 LEFT JOIN d.tags t
                 JOIN t.category c
             GROUP BY c.id
             """)
-    List<Pair> getAllOrdersForCategories();
+    List<Pair> getAllOrdersForCategories();>>>>>>> master
```

Comment:
Incorrect merge?

Reply:
I've resolved conflicts using github. I removed this.